src/test/java/org/opensearch/knn/quantization/output/BinaryQuantizationOutputTests.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.knn.quantization.output;
+
+import org.junit.Before;
+import org.opensearch.knn.KNNTestCase;
+import org.opensearch.knn.quantization.models.quantizationOutput.BinaryQuantizationOutput;
+
+public class BinaryQuantizationOutputTests extends KNNTestCase {
+
+    private static final int BITS_PER_COORDINATE = 1;
+    private BinaryQuantizationOutput quantizationOutput;
+
+    @Before
+    public void setUp() throws Exception {
+        super.setUp();
+        quantizationOutput = new BinaryQuantizationOutput(BITS_PER_COORDINATE);
```

Comment:
This member variable seems to be creating interdependency between tests, can we create this for every test please